frontend/packages/ux-editor/src/components/config/editModal/EditOptions/EditCodeList/EditCodeList.test.tsx

```diff
@@ -130,6 +131,54 @@ describe('EditCodeList', () => {
       await screen.findByText(textMock('ux_editor.modal_properties_error_message')),
     ).toBeInTheDocument();
   });
+
+  it('should render success toast if file upload is successful', async () => {
+    const user = userEvent.setup();
+    const file = new File(['hello'], 'hello.json', { type: 'text/json' });
+    await render({
+      queries: {
+        getOptionListIds: jest
+          .fn()
+          .mockImplementation(() => Promise.resolve<string[]>(optionListIdsMock)),
+      },
+    });
+
+    const btn = screen.getByText(textMock('ux_editor.modal_properties_code_list_upload'));
```

Comment:
Fine tester! Hva skjer hvis du bruker `getByRole` her?
```suggestion
    const btn = screen.getByRole('button', { name: textMock('ux_editor.modal_properties_code_list_upload') });
```
Du kan gjerne gjøre dette til en egen funksjon siden den brukes i begge testene.

Reply:
Fungerer fint, takk for hintet :smile: